src/frontend/dash/circlePacker.ts

```diff
@@ -30,101 +38,149 @@ export class MemberCircle {
     }
 }
 
-function distance(a, b) {
-    return Math.sqrt(Math.pow(b.x - a.x, 2) + Math.pow(b.y - a.y, 2))
-}
-function isColliding(a, b, margin) {
-    if (!margin) { margin = 0 }
-    return distance(a, b) < a.r + b.r + margin
-}
-function getCollided(circle, margin) {
-    for (const placedCircle of placedCircles) {
-        if (isColliding(circle, placedCircle, margin)) { return placedCircle }
-    }
-    return null;
+export function setAspectRatio(ratio : number) {
+    aspectRatio = ratio;
 }
 
-function placeCircleFromPoint(circle, pointX, pointY, angle, distance) {
-    circle.x = pointX + distance * Math.cos(angle) //rads
-    circle.y = pointY + distance * Math.sin(angle)
-}
-function getAngleFromAToB(a, b) {
-    // dot = a.x*b.x + a.y*b.y      
-    // det = a.x*b.y - a.y*b.x    
-    // return Math.atan2(det, dot) 
-    return 2 * Math.atan((b.y - a.y) / ((b.x - a.x) + Math.sqrt((b.x - a.x) * (b.x - a.x) + (b.y - a.y) * (b.y - a.y))))
+function getDistanceFrom(circle1, x, y) {
+    return Math.sqrt(Math.pow(circle1.x - x, 2) + Math.pow(circle1.y - y, 2));
 }
+
 function placeCircle(circle) {
-    // random starting position
-    const angle = Math.random() * 2 * Math.PI - Math.PI
-    let d = Math.max(maxX, Math.max(Math.abs(minX), Math.max(maxY, Math.abs(minY)))) + circle.r
-    placeCircleFromPoint(circle, 0, 0, angle, d)
-
-    // move inwards until touching
-    let collided = null
-    while (d > 0 && !collided) {
-        collided = getCollided(circle, MARGIN);
-        if (collided || d <= 0) { break }
-        d -= STEP_DISTANCE;
-        placeCircleFromPoint(circle, 0, 0, angle, d)
-        // redrawCircles(placedCircles.concat([circle])) // REDRAWWWWW = = = == = = = == = == = = = == 
-        // await sleep(10) /// REDRAWWWWW
+    // TODO: remove from placeCircle and onto placeCircles
```

Comment:
Is this old or something still to be done with this PR

Reply:
that's old. I need to fix that real quick